<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTMainWindow</class>
 <widget class="QMainWindow" name="RTMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RuckTrack</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/rucktrack.qrc">
    <normaloff>:/resources/icon.png</normaloff>:/resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowMap"/>
    <addaction name="actionShowTrackDetails"/>
    <addaction name="actionShowStatistics"/>
    <addaction name="actionShowGraphs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionFixElevations"/>
  </widget>
  <widget class="QToolBar" name="windowToolBar">
   <property name="windowTitle">
    <string>Window toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowMap"/>
   <addaction name="actionShowTrackDetails"/>
   <addaction name="actionShowStatistics"/>
   <addaction name="actionShowGraphs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="trackDetailsDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowIcon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/table.png</normaloff>:/resources/table.png</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Track details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="trackPointsView">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="mapDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_12">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="MapView" name="mapView" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statsDock">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>102</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Statistics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="StatisticsTable" name="statsTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="graphsDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Graphs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="graphXCombo">
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="graphY1Combo">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Elevation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Speed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Climb rate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Total climb</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="graphY2Combo">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Elevation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Speed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Climb rate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Total climb</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Display:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="graphDisplayCombo">
         <item>
          <property name="text">
           <string>Segmented</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Continuous</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="PlotView" name="plotView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/document-open.png</normaloff>:/resources/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShowMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/map.png</normaloff>:/resources/map.png</iconset>
   </property>
   <property name="text">
    <string>Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShowTrackDetails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/table.png</normaloff>:/resources/table.png</iconset>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
   <property name="toolTip">
    <string>Show track details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionShowStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/view-table-of-contents-ltr.png</normaloff>:/resources/view-table-of-contents-ltr.png</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/document-save.png</normaloff>:/resources/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShowGraphs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/office-chart-area.png</normaloff>:/resources/office-chart-area.png</iconset>
   </property>
   <property name="text">
    <string>Graphs</string>
   </property>
   <property name="toolTip">
    <string>Show graphs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionFixElevations">
   <property name="icon">
    <iconset resource="../resources/rucktrack.qrc">
     <normaloff>:/resources/tools-wizard.png</normaloff>:/resources/tools-wizard.png</iconset>
   </property>
   <property name="text">
    <string>Fix elevations</string>
   </property>
   <property name="toolTip">
    <string>Calculate precise elevation values from SRTM data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StatisticsTable</class>
   <extends>QTableWidget</extends>
   <header>StatisticsTable.h</header>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QWidget</extends>
   <header>MapView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotView</class>
   <extends>QWidget</extends>
   <header>PlotView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/rucktrack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionShowTrackDetails</sender>
   <signal>toggled(bool)</signal>
   <receiver>trackDetailsDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowGraphs</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphsDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowStatistics</sender>
   <signal>toggled(bool)</signal>
   <receiver>statsDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowMap</sender>
   <signal>toggled(bool)</signal>
   <receiver>mapDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowGraphs</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphsDock</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowTrackDetails</sender>
   <signal>toggled(bool)</signal>
   <receiver>trackDetailsDock</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowMap</sender>
   <signal>toggled(bool)</signal>
   <receiver>mapDock</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowStatistics</sender>
   <signal>toggled(bool)</signal>
   <receiver>statsDock</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
